test/e2e/arktos/kubectl/config-kubectl-e2e-test.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Copyright 2020 Authors of Arktos.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+base_dir=$(cd $(dirname $0)/../../../.. ; pwd)
+
+# By default we use the kubectl binary built in the Arktos repository.
+kubectl=${base_dir}/_output/bin/kubectl
```

Comment:
maybe also check an env variable and if it's not empty, use that.

Reply:
It may look weird I just allow the variable "kubectl" to take a value of env vars. If I make changes to all the variables, that will break my original intention. 

I will leave it as is for now. If some user complain about it, I will update.